modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -266,20 +266,30 @@ a|Write the ISO to a new output file.
 a|`-h`, `--help`
 a|Print help information.
 
+a|
+a|
+
 2+|_coreos-installer PXE Ignition commands_
 
 |*Command* |*Description*
 
-a|`coreos-installer pxe ignition wrap <options> <initrd_name>`
-a|Wrap an Ignition config in an `initrd` image.
+2+|Note that not all of these options are accepted by all subcommands.
 
-a|`coreos-installer pxe ignition unwrap <options> <initrd_name>`
-a|Show the wrapped Ignition config in an `initrd` image.
+a|`coreos-installer pxe ignition wrap <options> <image_name>`
```

Comment:
`wrap` doesn't take `<image_name>`.

Reply:
Changed to `coreos-installer pxe ignition wrap <options>`

Reply:
@bgilbert To clarify, does `unwrap` take `<image_name>` though? 

In the next line, we have `coreos-installer pxe ignition unwrap <options> <image_name>`. I'm going to leave it that way, but let me know if `<image_name>` should be removed.

Reply:
Yes, `unwrap` takes an image name but `wrap` does not.

Reply:
Perfect, thanks!